internal/indexcoord/meta_table.go

```diff
@@ -42,8 +42,8 @@ import (
 // revision: The number of times IndexMeta has been changed in etcd. It's the same as Event.Kv.Version in etcd.
 // indexMeta: A structure that records the state of the index defined by proto.
 type Meta struct {
-	indexMeta *indexpb.IndexMeta
-	revision  int64
+	indexMeta    *indexpb.IndexMeta
+	indexVersion int64
```

Comment:
Misunderstanding. It will be changed in the next PR. The `revision` here corresponds to the version of the key in etcd.
What you want to change is actually the Version in indexpb.